public/locales/es.json

```diff
@@ -47,7 +47,7 @@
   "Add a comment": "Agregar un comentario",
   "Add a comment...": "Agregar un comentario...",
   "Add a description": "Agregar una descripción",
-  "Add a description, location, suggested topics and more in your group settings": "Agregar una descripción, ubicación, temas sugeridos y más en la configuración de tu grupo",
+  "Add a description, location, suggested topics and more in your group settings": "",
```

Comment:
why is this blank?

Reply:
Since this merge my i18n has been acting funny, I assume a tweak to the configuration somehow. Some version of this has happened before when it somehow nukes translations automagically :(